src/pages/org/Orgs.jsx

```diff
@@ -1,28 +1,19 @@
 import React, { useState } from 'react';
-import { useSelector } from 'react-redux';
 import { useIntl } from 'react-intl';
 import { get } from 'lodash-es';
 import MainColumn from '../../components/MainColumn';
 import AvatarGallery from '../../components/AvatarGallery';
 import ConfirmDelete from '../../components/ConfirmDelete';
 import Header from '../../components/Header';
 import Text from '../../components/Text';
-import { selectOrgs } from '../../modules/orgs/selectors';
-import { selectSiteName } from '../../modules/site/selectors';
-import { selectIsAdministrator } from '../../modules/app/selectors';
 import useDocumentTitle from '../../hooks/useDocumentTitle';
 
 export default function User() {
   const intl = useIntl();
-  const siteName = useSelector(selectSiteName);
-  const pageTitle = intl.formatMessage(
-    { id: 'ORGS_PAGE_TITLE' },
-    { siteName },
-  );
-  useDocumentTitle(pageTitle, { translateMessage: false });
+  useDocumentTitle('ORGANIZATIONS');
 
-  const orgs = useSelector(selectOrgs);
-  const isAdministrator = useSelector(selectIsAdministrator);
+  const orgs = [];
+  const isAdministrator = true;
```

Comment:
This might become easy to lose track of. Maybe we can add a TODO comment here or connect this to a config file or something? 

In other words, I presume that we won't always want this to be true forever, but I'm  not sure what we should do to make sure this doesn't become a bug.

Reply:
Thing is the orgs UI isn't even part of MVP. So this whole file is more or less TODO status. Along with projects